src/metabase/api/recent_errors.clj

```diff
@@ -0,0 +1,25 @@
+(ns metabase.api.recent-errors
+  (:require [amalloy.ring-buffer :as ring-buffer]
+            [compojure.core :refer [GET]]
+            [metabase.api.common :as api]))
+
+(set! *warn-on-reflection* true)
+
+(def ^:private recent-errors (atom (ring-buffer/ring-buffer 10)))
+
+(defn record!
```

Comment:
Is this called anywhere?

Reply:
Nope - we should delete this entire namespace. @iethree would you be able to rebase -i and remove all my commits except for https://github.com/metabase/metabase/pull/39817/commits/25d53dc9dfc2021d35efca190bd4aeafbaa6135b ?